hazelcast-ra/hazelcast-jca/src/test/java/com/hazelcast/jca/XATestWithJCA.java

```diff
@@ -79,6 +88,69 @@ public void testPut() throws ResourceException {
         assertEquals(val, m.get(key));
     }
 
+    @Test
+    public void testTransactionalQueueShouldPollWhatWasOffered() throws ResourceException {
+        String name = randomString();
+        String item = randomString();
+
+        TransactionalQueue<String> queue = connection.getTransactionalQueue(name);
+        queue.offer(item);
+
+        connection.close();
```

Comment:
is there any specific reason not to close the connection within teardown method annotated with `@After`?